### PR TITLE
backup script & image change

### PR DESCRIPTION
**Pull request by ozgurcd**
Backup script has been modified
Image for watcherdb-backup container has been changed

### Motivation

necessary changes to separate S3 backups on a sidecar container

### CODE REVIEW

File: .internal-ci/helm/watcher/templates/watcher-backup-scripts.yaml [hunk 1]
```diff
@@ -11,12 +11,15 @@ data:
...
-    SECOND=$(date '+%S')
+    MDB_STAT="/usr/bin/mdb_stat"
+    EPOCH=`date '+%s'`
+
+    YEAR=$(date -d@$EPOCH '+%Y')
+    MONTH=$(date -d@$EPOCH '+%m')
+    DAY=$(date -d@$EPOCH '+%d')
+    HOUR=$(date -d@$EPOCH '+%H')
+    MINUTE=$(date -d@$EPOCH '+%M')
+    SECOND=$(date -d@$EPOCH '+%S')
```
Comment (jgreat) [inline .internal-ci/helm/watcher/templates/watcher-backup-scripts.yaml:22]: Do we need this script at all if we're using the new image.  Don't we have a copy in that repo?

File: .internal-ci/helm/watcher/templates/watcher-backup-scripts.yaml [hunk 2]
```diff
@@ -49,12 +52,29 @@ data:
...
 
+    # Create the copy directory
+    mkdir -p /watcher-backup
+
     # Copy the db file for stable upload
-    cp /watcher/data.mdb /tmp/data.mdb
+    mdb_copy /watcher /watcher-backup
+
+    cd /watcher-backup
+    $MDB_STAT -s watcher_db:block_signatures . 2> /dev/null
```
Comment (joekottke) [inline .internal-ci/helm/watcher/templates/watcher-backup-scripts.yaml:62]: Whats the reason you reference `mdb_stat` with a variable and not `mdb_copy`?   I think either is fine, but we should be consistent.

File: .internal-ci/helm/watcher/templates/watcher-backup-scripts.yaml [hunk 3]
```diff
@@ -49,12 +52,29 @@ data:
...
+    if [ $? -ne 0 ]
+    then
+            echo "Bad backup, giving up"
+            # SEND SLACK NOTICE
```
Comment (joekottke) [inline .internal-ci/helm/watcher/templates/watcher-backup-scripts.yaml:66]: Maybe a TODO here?

File: .internal-ci/helm/watcher/templates/watcher-backup-scripts.yaml [hunk 4]
```diff
@@ -11,12 +11,15 @@ data:
...
 
-    YEAR=$(date '+%Y')
-    MONTH=$(date '+%m')
-    DAY=$(date '+%d')
-    HOUR=$(date '+%H')
-    MINUTE=$(date '+%M')
-    SECOND=$(date '+%S')
+    MDB_STAT="/usr/bin/mdb_stat"
+    EPOCH=`date '+%s'`
+
```
Comment (joekottke) [inline .internal-ci/helm/watcher/templates/watcher-backup-scripts.yaml:16]: Smart!

**Review by joekottke — APPROVED** [review]
Nothing stopping the approval, just some nits and curiosities.

**Comment by jcape** [comment]
@ozgurcd Only thing that needs doing is resolving conversations (i.e. if you just want to merge it, you can just resolve the conversations, if you want to fix things, push the fixes, then resolve).

**Comment by jcape** [comment]
@ozgurcd Should this be merged, or closed?

**Comment by ozgurcd** [comment]
> 
@jgreat tookover it as far as I could remember,. @jgreat  what do you say?

**Pull request closed without merging**